frontend/app/sysupport/detail/index/template.hbs

```diff
@@ -0,0 +1,62 @@
+<div uk-grid class="uk-child-width-1-1 uk-child-width-1-2@m" uk-scrollspy="target: > div; cls:uk-animation-slide-bottom-small; delay: 100">
+  <div>
+    {{#uk-card as |card|}}
+      {{#card.header}}
+        {{#card.title}}
+          {{t 'sysupport.detail.expense'}}
+        {{/card.title}}
+      {{/card.header}}
+      {{#card.body}}
+        <table class="uk-table uk-table-divider uk-width-1-1">
+          <thead>
+            <tr>
+              <th>{{t 'sysupport.detail.date'}}</th>
+              <th>{{t 'sysupport.detail.effort'}}</th>
+              <th>{{t 'sysupport.detail.employee'}}</th>
+              <th>{{t 'sysupport.detail.description'}}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {{#each model.reports as |report index|}}
+              <tr data-test-project-report="{{index}}">
+                <td>{{moment-format report.date 'DD.MM.YYYY'}}</td>
+                <td>{{format-duration-short report.duration}}</td>
+                <td>{{report.user.fullName}}</td>
+                <td>{{report.comment}}</td>
+              </tr>
+            {{else}}<tr><td>{{t 'global.empty'}}</td></tr>{{/each}}
```

Comment:
`<td>` needs a colspan or it will look ugly

Reply:
:+1: